explainers/subresource-loading.md

```diff
@@ -206,10 +206,18 @@ of resources could look like:
 <link
   rel="webbundle"
   href="https://example.com/dir/subresources.wbn"
-  resources="https://example.com/this,https://example.com/is,https://example.com/not,https://example.com/a,https://example.com/real,https://example.com/list,https://example.com/of,https://example.com/URLs"
+  resources="https://example.com/this https://example.com/is https://example.com/not https://example.com/a https://example.com/real https://example.com/list https://example.com/of https://example.com/URLs"
 />
 ```
 
+IDL:
+
+``` webidl
```

Comment:
We're not supposed to put WebIDL into explainers at all. You could show this as some sample Typescript code using the link element's object?

Reply:
Ah, I didn't know that. Thanks!
Regarding a TypeScript example, I am not sure we could have a good example here. So let's just drop the IDL.

Let me update the commit message, and rebase it.